client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -17,13 +17,15 @@ type Props = ExternalProps & LocalizeProps;
 
 export const WarningList = ( { context, translate, warnings, showContact = true }: Props ) => (
 	<div>
-		<div className="eligibility-warnings__warning">
-			<div className="eligibility-warnings__message">
-				<span className="eligibility-warnings__message-description">
-					{ getWarningDescription( context, warnings.length, translate ) }
-				</span>
+		{ getWarningDescription( context, warnings.length, translate ) && (
```

Comment:
To avoid rendering a void box with a border-bottom line on it, we only render if we get something from `getWarningDescription`